commit 79409f85f3c02f53ef267647e2bdf2f9a6cf515e
Author: John Levon <john.levon@joyent.com>
Date:   2019-07-05T13:21:31+00:00 (3 months ago)
    
    TRITON-1795 remove cruft from sdcadm's post-setup tests
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/test/post-setup.test.js b/test/post-setup.test.js
index 08e9b45..80dbba9 100644
--- a/test/post-setup.test.js
+++ b/test/post-setup.test.js
@@ -5,14 +5,12 @@
  */
 
 /*
- * Copyright 2018, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
- * PENDING TESTS:
+ * Missing tests for:
  *
- * - Complete ha-manatee setup
- * - Complete ha-bider setup
  * - post-setup fabrics
  * - post-setup underlay-nics
  */
@@ -320,38 +318,6 @@ test('sdcadm post-setup help dev-sample-data', function (t) {
 });
 
 
-test('sdcadm post-setup ha-manatee', function (t) {
-    exec('sdcadm post-setup ha-manatee', function (err, stdout, stderr) {
-        t.ok(err);
-
-        t.equal(stdout, '');
-        t.notEqual(stderr.indexOf('Must specify 2 target servers'), -1);
-
-        t.end();
-    });
-});
-
-
-test('sdcadm post-setup ha-manatee --servers', function (t) {
-    var serverUuids = '';
-
-    exec('sdcadm post-setup ha-manatee -s' + serverUuids,
-         function (err, stdout, stderr) {
-             console.log(err);
-             console.log(stdout);
-             console.log(stderr);
-        // TODO
-        t.end();
-    });
-});
-
-
-test('sdcadm post-setup help ha-manatee', function (t) {
-    checkHelp(t, 'post-setup ha-manatee',
-              'Create 2nd and 3rd manatee instances');
-});
-
-
 test('sdcadm post-setup help fabrics', function (t) {
     checkHelp(t, 'post-setup fabrics',
               'Create portolan instance and setup fabrics');
